README.md

```diff
@@ -1 +1,176 @@
+
+<details>
+  <summary>CSS3</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/CSS">Info CSS3</a></li>
+  </ul>
+</details>
+
+### Tech Stack <a name="tech-stack">JS</a>
+
+<details>
+  <summary>JS</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/JavaScript">Info JS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Set up the linter for fixing the errors and clean code**
+- **Add the structured content using HTML**
+- **Add the styling for the content using CSS**
+- **Demonstrate ability to create UIs adaptable to different screen sizes using media queries**
+- **Add functions for make interactive content using JS**
+- **Use JavaScript to manipulate DOM elements**
+- **Use objects to store and access data**
+- **Work with Flexbox properties in the CSS**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## Video Presentation <a name="video-presentation"></a>
+
+- [Video Link](https://www.loom.com/share/bb83e63a1cb745f1ad2edf1aa6672dee)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://biancame.github.io/Capstone/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+A computer
+
+### Setup
+
+Clone this repository to your desired folder:
+
+cd my-folder
+git clone https://github.com/BiancaMe/CapstoneProject.git
+
+### Usage
+
+To run the project, double click on the index to open it in the browser.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Bianca Lopez**
+
+- GitHub: [@BiancaMe](https://github.com/BiancaMe)
+- Twitter: [@BiancaMeLopez](https://twitter.com/BiancaMeLopez)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/bianca-lopez-55a4a3276/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add a market for buy **
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can contac me and let me know.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thanks Microverse for give the oportunity to increase my knowledge
+
```

Comment:
- [ OPTIONAL ] Good job working on the README so far but as mentioned in the requirements you should give the appropriate acknowledgment to the credit to the author. so kindly add it here  🙏🏻🎉⭐ I am assigning to optional because the previous reviewer didn't mention it but it is still important to fix it 🙏🏻

![image](https://github.com/BiancaMe/Capstone/assets/58553711/40616184-bb23-4639-86cb-b0e6168ef15c)